starters/solidjs-tailwind/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "description": "SolidJS and TailwindCSS",
+  "author": "Maarten, Chinedu, Daina, Mattai, Mark, Victor, Dane, Jan, Jerry",
```

Comment:
Please change Mattai to Mattia 🤣 🙏🏻

Reply:
> Please change Mattai to Mattia 🤣 🙏🏻

Why should i @Megio ? I prefer mine 🤣 . I agree with @mbicknese